config/kibana.yml

```diff
@@ -44,8 +44,8 @@
 # the username and password that the Kibana server uses to perform maintenance on the Kibana
 # index at startup. Your Kibana users still need to authenticate with Elasticsearch, which
 # is proxied through the Kibana server.
-#elasticsearch.username: "kibana"
-#elasticsearch.password: "pass"
+elasticsearch.username: "elastic"
```

Comment:
your local config changes sneaked in

Reply:
Oops! Thanks.

Reply:
@cachedout See https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#customizing-configkibanadevyml